noir/noir-repo/tooling/debugger/src/context.rs

```diff
@@ -551,16 +551,16 @@ impl<'a, B: BlackBoxFunctionSolver> DebugContext<'a, B> {
         let opcodes = self.get_opcodes();
         match *location {
             OpcodeLocation::Acir(acir_index) => acir_index < opcodes.len(),
-            OpcodeLocation::Brillig { acir_index, brillig_index } => {
+            OpcodeLocation::Brillig { acir_index, .. } => {
                 acir_index < opcodes.len()
-                    && matches!(opcodes[acir_index], Opcode::Brillig(..))
-                    && {
-                        if let Opcode::Brillig(ref brillig) = opcodes[acir_index] {
-                            brillig_index < brillig.bytecode.len()
-                        } else {
-                            false
-                        }
-                    }
+                    && matches!(opcodes[acir_index], Opcode::BrilligCall { .. })
+                // && {
+                //     if let Opcode::Brillig(ref brillig) = opcodes[acir_index] {
+                //         brillig_index < brillig.bytecode.len()
+                //     } else {
+                //         false
+                //     }
+                // }
```

Comment:
```suggestion
```
If these snippets are a TODO for remaining work we should make that more explicit